docs/_data/sidebars/pmd_sidebar.yml

```diff
@@ -94,9 +94,6 @@ entries:
         - title: Defining rule properties
           url: /pmd_userdocs_extending_defining_properties.html
           output: web, pdf
-        - title: Using and defining code metrics
```

Comment:
Yes that's true, I think we could give pointers to the javadocs in the guide page for java rules. 

I think eventually we should also make a page to collect info about pmd-java specific features (eg some doc about the symbol table, the typeres api, etc). This would also be useful for apex, as the module gets more complex (#2667). Those pages could also have a section about metrics.

Reply:
I created a draft for the java page, I don't think it should block this PR. (But I still have to add info about, well, metrics....)

Reply:
I think I'm done, I'll open a ticket so we don't forget to finish writing those pages